infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -22,263 +22,251 @@ cloud_run_config = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://asia.gcr.io"
+        value = "https://asia-east1-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   asia-northeast1 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://asia.gcr.io"
+        value = "https://asia-northeast1-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   asia-northeast2 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://asia.gcr.io"
+        value = "https://asia-northeast2-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   asia-south1 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://asia.gcr.io"
+        value = "https://asia-south1-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   asia-southeast1 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://asia.gcr.io"
+        value = "https://asia-southeast1-docker.pkg.dev"
```

Comment:
$ crane pull asia-southeast1-docker.pkg.dev/k8s-artifacts-prod/images/pause:3.2 /dev/null && echo $?
2022/10/12 23:34:44 No matching credentials were found for "asia-southeast1-docker.pkg.dev"
Error: pulling asia-southeast1-docker.pkg.dev/k8s-artifacts-prod/images/pause:3.2: GET https://asia-southeast1-docker.pkg.dev/v2/token?scope=repository%3Ak8s-artifacts-prod%2Fimages%2Fpause%3Apull&service=asia-southeast1-docker.pkg.dev: DENIED: Permission "artifactregistry.repositories.downloadArtifacts" denied on resource "projects/k8s-artifacts-prod/locations/asia-southeast1/repositories/images" (or it may not exist)